kong/pdk/response.lua

```diff
@@ -348,11 +348,8 @@ local function new(self, major_version)
   -- Allows changing the downstream response HTTP status code before sending it
   -- to the client.
   --
-  -- This function should be used in the `header_filter` phase, as Kong is
-  -- preparing headers to be sent back to the client.
-  --
   -- @function kong.response.set_status
-  -- @phases rewrite, access, header_filter, admin_api
+  -- @phases rewrite, access, header_filter, response, admin_api
```

Comment:
Same question here, will `set_status` work in `response` phase?

Reply:
i think it should, as neither body nor headers have been sent to the client